src/components/CMessageRow.vue

```diff
@@ -99,15 +99,17 @@ export default {
         return null;
       }
       const factor = (1 - this.progress) / this.progress;
-      const diff = utils.formatMillis((new Date() - this.startedDatetime) * factor);
+      const diff = utils.formatMillis(
```

Comment:
what is the timezone of this.startedDatetime here ? you cannot compare 2 dates as you do with different timezones.

Reply:
umm I think it is UTC,  [started date timezone](https://github.com/wiremind/remoulade/blob/master/remoulade/state/middleware.py), I just change to UTC when is instanced new Date()